internal/sql/pg/query.go

```diff
@@ -180,6 +180,7 @@ func query(ctx context.Context, oidToDataType map[uint32]*datatype, cq connQuery
 
 	// res := rows.CommandTag() // RowsAffected, bool for Select etc.
 	resSet := &sql.ResultSet{}
+	var oids []uint32
 	for _, colInfo := range rows.FieldDescriptions() {
 		// fmt.Println(colInfo.DataTypeOID, colInfo.DataTypeSize)
```

Comment:
remove this comment.

Reply:
Will do in follow up that's touching this code